lib/Items/items.h

```diff
@@ -13,9 +13,9 @@ enum ItemType
 class Item
 {
 public:
+    Item();
```

Comment:
is there a need for an empty item object?

Reply:
yes there is, as we need to make an array of items, the items need to be initialized at the time of creating the array.

Reply:
ah i see